src/graph/graph.c

```diff
@@ -745,8 +749,10 @@ int Graph_DeleteEdge(Graph *g, Edge *e) {
 		if(TR) {
 			/* We must make the matching updates to the transposed matrix.
 			 * First, extract the element that is known to be an edge array. */
-			info = GrB_Matrix_extractElement(edges, TR, dest_id, src_id);
+			info = GrB_Matrix_extractElement(&edge_id, TR, dest_id, src_id);
 			assert(info == GrB_SUCCESS);
+			edges = (EdgeID *)edge_id;
+
```

Comment:
`/* First, extract the element that is known to be an edge array. */`
either change to comment or revert the change

also
`edges = (EdgeID *)edge_id;`
is set at line 728